client/package.json

```diff
@@ -46,6 +47,7 @@
     "axios": "^0.17.1",
     "bootstrap": "^3.3.7",
     "js-cookie": "^2.2.0",
+    "moment": "^2.20.1",
```

Comment:
I see moment implemented, but not used anywhere?

Reply:
soon... it will be used in sources to show most recent scan. I can remove for now and add back if you like.

Reply:
That works, lets go for it